packages/airnode-utilities/src/evm/gas-prices/gas-oracle.test.ts

```diff
@@ -293,6 +301,69 @@ describe('Gas oracle', () => {
       expect(gasTarget).toEqual(gasOracle.getGasTargetWithGasLimit(providerRecommendedGasPrice, fulfillmentGasLimit));
     });
 
+    it('returns baseFeePerGas if getGasPrice too high', async () => {
+      const getGasPriceSpy = jest.spyOn(ethers.providers.StaticJsonRpcProvider.prototype, 'getGasPrice');
+      const getGasPriceMock = ethers.BigNumber.from(1000);
+      getGasPriceSpy.mockImplementation(() => Promise.resolve(getGasPriceMock));
+
+      const fetchBaseFeePerGasMock = jest.spyOn(gasOracle, 'fetchBaseFeePerGas');
+      const baseFeePerGasMock = ethers.BigNumber.from(1);
+      fetchBaseFeePerGasMock.mockResolvedValue(baseFeePerGasMock);
+
+      const providerRecommendedGasPrice = await gasOracle.fetchProviderRecommendedGasPrice(
+        provider,
+        providerRecommendedGasPriceStrategy,
+        startTime
+      );
+
+      const expectedGasTarget = baseFeePerGasMock.mul(2).add(3);
+      expect(providerRecommendedGasPrice.gasPrice).toEqual(expectedGasTarget);
+    });
+
+    it('returns getGasPrice if not too high', async () => {
```

Comment:
"not too high" is a big vague here. Perhaps something along the lines of `returns getGasPrice if the multiplied gas price is less than a multiple of the baseFeePerGas`